metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/dereference/Dereferencer.java

```diff
@@ -21,7 +22,7 @@
    * @return A list of RDF field names with the information associated with it
    * @throws DereferenceException
    */
-  List<EnrichmentBase> dereferenceEntities(Set<String> resourceIds) throws DereferenceException;
+  List<EnrichmentBase> dereferenceEntities(Set<ReferenceTerm> resourceIds) throws DereferenceException;
```

Comment:
The main reason why I changed to `Set<ReferenceTerm>` was because in the implementation, it would eventually need to change to `Set<ReferenceTerm>` as an input for anothr method.
But indeed, this is the Interface and the url is the most important information for this method. Will revert this